torch/_dynamo/variables/constant.py

```diff
@@ -81,6 +81,12 @@ def unpack_var_sequence(self, tx):
             raise NotImplementedError from e
 
     def const_getattr(self, tx, name):
+        if isinstance(self.value, type):
+            raise UserError(
+                UserErrorType.ANTI_PATTERN,
+                "Can't access members of type(obj) for a generated custom object. "
+                "Please use __class__ instead",
+            )
```

Comment:
I still don't really understand this error. Is this a notimplemented in disguise?

Reply:
I think so. I just pushed it further, as suggested [in this comment](https://github.com/pytorch/pytorch/pull/98366/files#r1159296709).